LiveSplit.AutoSplitters.xml

```diff
@@ -1,6 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <AutoSplitters>
     <AutoSplitter>
+        
+        
```

Comment:
What happened here?

Reply:
Oh, apologies. I was originally going to add it to the top, then realised I wanted to edit a similar game and added it alongside it. I completely forgot to go back and remove the extra lines I'd added at the top.